app/tests/integration/nimbus/conftest.py

```diff
@@ -414,6 +415,12 @@ def _trigger_experiment_loader():
                     ChromeUtils.import(
                         "resource://nimbus/lib/RemoteSettingsExperimentLoader.jsm"
                     ).RemoteSettingsExperimentLoader.updateRecipes();
+                    
+                    const { RemoteSettings } = ChromeUtils.import("resource://services-settings/remote-settings.js");
+                    const { RemoteSettingsExperimentLoader } = ChromeUtils.import("resource://nimbus/lib/RemoteSettingsExperimentLoader.jsm");
+
+                    RemoteSettings.pollChanges();
+                    RemoteSettingsExperimentLoader.updateRecipes();
```

Comment:
Oh is this the magic fix?

Reply:
yup!

Reply:
You don't need the first call to updateRecipes() on line 417